java/search-for-webmap/src/main/res/layout/activity_main.xml

```diff
@@ -18,14 +18,15 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:gravity="center"
-        android:text="Use the search bar at the top to search for web maps" />
+        android:text="@string/to_search_text" />
 
     <android.support.v7.widget.RecyclerView
         android:id="@+id/webmapRecyclerView"
         android:layout_width="240dp"
         android:layout_height="match_parent"
         android:layout_gravity="start"
-        android:background="@color/drawerBackground"/>
+        android:background="@color/drawerBackground"
+        android:scrollbars="vertical" />
```

Comment:
Is this necessary?

Reply:
It was one of the feedback suggestions for the samples team to give an indication of where on the list the user is.

Reply:
Sorry, missed that as a requirement.